corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/parcReader/PARC3Reader.java

```diff
@@ -30,61 +31,66 @@
 
 /**
  * A Reader for Penn Attribution Relations Copurs 3.0 (PARC 3.0)
-
- * Documents in corpus are stored as a syntactic tree in XML. Annotations are stored as child nodes to "Word" nodes
-
+ * 
+ * Documents in corpus are stored as a syntactic tree in XML. Annotations are stored as child nodes
+ * to "Word" nodes
+ * 
  * A toy example document below:
-
+ * 
  * <root>
-
- *     <SENTENCE gorn="0">
-
- *          <NP-HLN gorn="0">
-
- *              <WORD ByteCount="9,16" gorn="0,0" lemma="two-way" pos="NNP" sentenceWord="0" text="Two-Way" word="0">
-
- *                  <attribution id="wsj_2401_PDTB_annotation_level.xml_set_0">
-
- *                      <attributionRole roleValue="content"/>
-
- *                  </attribution>
-
- *              </WORD>
-
- *              <WORD ByteCount="17,23" gorn="0,1" lemma="street" pos="NNP" sentenceWord="1" text="Street" word="1"/>
-
- *          </NP-HLN>
-
- *      </SENTENCE>
-
+ * 
+ * <SENTENCE gorn="0">
+ * 
+ * <NP-HLN gorn="0">
+ * 
+ * <WORD ByteCount="9,16" gorn="0,0" lemma="two-way" pos="NNP" sentenceWord="0" text="Two-Way"
+ * word="0">
+ * 
+ * <attribution id="wsj_2401_PDTB_annotation_level.xml_set_0">
+ * 
+ * <attributionRole roleValue="content"/>
+ * 
+ * </attribution>
```

Comment:
This still looks "double-spaced" -- right now:
``` 
    * <attribution... >...
    * 
    * <attributionRole....> ...
```
rather than:
``` 
    * <attribution... >... 
    * <attributionRole....> ...
```
-- does this make sense?